bundles/extensions/i18n/src/main/java/org/apache/sling/i18n/impl/JcrResourceBundleProvider.java

```diff
@@ -226,6 +257,14 @@ public void handleEvent(final org.osgi.service.event.Event event) {
         }
     }
 
+    private void discardBundlesWithLocalePrefix(String localePrefix) {
```

Comment:
@kwin IMO if there is a change in dictionary de_AT and we need to reload a bundle with locale de_AT all of its other variant like de_AT_MAC but we do not need to reload bundle with locale as de. 
More ever scheduling a bundle to reload will also reload parent bundle so I am not sure which case I am missing here. 
Can you please let me know on this ?

thanks,